homeassistant/components/binary_sensor/mercedesme.py

```diff
@@ -0,0 +1,107 @@
+"""
+Support for Mercedes cars with Mercedes ME.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/mercedesme/
+"""
+import logging
+import datetime
+
+from homeassistant.components.binary_sensor import (BinarySensorDevice)
+from homeassistant.components.mercedesme import (
+    DATA_MME, MercedesMeEntity, BINARY_SENSORS)
+
+DEPENDENCIES = ['mercedesme']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    data = hass.data[DATA_MME].data
+
+    if not data.cars:
+        _LOGGER.error("setup_platform data.cars is none")
+        return
+
+    devices = []
+    for car in data.cars:
+        for dev in BINARY_SENSORS:
+            devices.append(MercedesMEBinarySensor(
+                data, dev, dev, car["vin"], None))
+
+    add_devices(devices, True)
+
+
+class MercedesMEBinarySensor(MercedesMeEntity, BinarySensorDevice):
+    """Representation of a Sensor."""
+
+    @property
+    def is_on(self):
+        """Return the state of the binary sensor."""
+        retval = True if bool(self._state == "On") else False
+        return retval
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        return self._state
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        if self._name == "windowsClosed":
+            return {
+                "windowStatusFrontLeft": self._car["windowStatusFrontLeft"],
+                "windowStatusFrontRight": self._car["windowStatusFrontRight"],
+                "windowStatusRearLeft": self._car["windowStatusRearLeft"],
+                "windowStatusRearRight": self._car["windowStatusRearRight"],
+                "originalValue": self._car[self._name],
+                "lastUpdate": datetime.datetime.fromtimestamp(
+                    self._car["lastUpdate"]).strftime('%Y-%m-%d %H:%M:%S'),
+                "car": self._car["license"]
+            }
+        elif self._name == "tireWarningLight":
+            return {
+                "frontRightTirePressureKpa":
+                    self._car["frontRightTirePressureKpa"],
+                "frontLeftTirePressureKpa":
+                    self._car["frontLeftTirePressureKpa"],
+                "rearRightTirePressureKpa":
+                    self._car["rearRightTirePressureKpa"],
+                "rearLeftTirePressureKpa":
+                    self._car["rearLeftTirePressureKpa"],
+                "originalValue": self._car[self._name],
+                "lastUpdate": datetime.datetime.fromtimestamp(
+                    self._car["lastUpdate"]
+                    ).strftime('%Y-%m-%d %H:%M:%S'),
+                "car": self._car["license"],
+                "Is_On": self.is_on,

```

Comment:
This is already the binary sensor state and isn't necessary to include in the attributes.